src/utilities/renavam/index.ts

```diff
@@ -0,0 +1,43 @@
+import { onlyNumbers } from '../../helpers';
+
+export const MIN_LENGTH = 9;
+export const LENGTH = 11;
+export const SUM = 284;
+
+export const isValid = (input: string) => {
+  if (!input || typeof input !== 'string' || input.length < MIN_LENGTH) return false;
+
+  let result = onlyNumbers(input.padStart(11, '0'));
```

Comment:
You could use `const` and (if you think that make sense) rename `result` to `numeric`, just to be more explicit what it is.